File: java/src/main/java/com/ortecfinance/tasklist/TaskListApplication.java
package com.ortecfinance.tasklist;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class TaskListApplication {

    public static void main(String[] args) {
        if (args.length == 0) {
            System.out.println("Starting console Application");
            TaskList.startConsole();
        }
        else {
            SpringApplication.run(TaskListApplication.class, args);
            System.out.println("localhost:8080/tasks");
        }
    }

}

File: kotlin/src/main/kotlin/com/ortecfinance/tasklist/TaskListApplication.kt
package com.ortecfinance.tasklist

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class TaskListApplication

fun main(args: Array<String>) {
    if (args.isEmpty()) {
        println("Starting console Application")
        TaskList.startConsole()
    } else {
        runApplication<TaskListApplication>(*args)
        println("localhost:8080/tasks")
    }
}
